charts/hono/README.md

```diff
@@ -534,20 +534,24 @@ of Quarkus based services images.
 ## Using Kafka based Messaging
 
 The chart can be configured to use Kafka as the messaging network instead of an AMQP 1.0 messaging network.
-The property `messagingNetworkType` is used to select the type of the messaging network.
+The configuration `messagingNetworkTypes[0]=kafka` deploys Hono configured to use Kafka for messaging.
+It is possible to enable AMQP _and_ Kafka based messaging at the same time (command line parameters of the deployment 
+would be `--set messagingNetworkTypes[0]=amqp --set messagingNetworkTypes[1]=kafka`). Each tenant in Hono can then be 
```

Comment:
How about using `--set "messagingNetworkTypes={amqp,kafka}"` instead?

Reply:
This syntax is more error-prone FMPOV. If you forget to quote the property, it will fail on the command line. All other command line parameters in this document do not need quotation.

Reply:
yes, ok.